### ISSUE TITLE
Can darknet build with Debug in vs?

### ISSUE THREAD

**Issue by hfq0219** [AlexeyAB/darknet#6401]
I can't build darknet with Debug in vs, and it shows 
"c:\program files\nvidia gpu computing toolkit\cuda\v10.0\include\crt/common_functions.h(150): error : more than one instance of overloaded function "_free_dbg" has "C" linkage"
![捕获](https://user-images.githubusercontent.com/17703923/89173898-88424f80-d5b7-11ea-99e1-7245b495106c.PNG)

**Comment by Lucky466** [comment]
of course, just use cmake compile darknet,then open project, install dependencis(opencv3.4.11 has been proved worked), then you can enter into debug
